src/interfaces/swap-asset.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { Trade } from 'swap-router-sdk';
+
+import { AssetAmountInterface } from '../components/asset-amount-input/asset-amount-input';
+
+export interface SwapFormValues {
+  inputAssets: AssetAmountInterface;
+  outputAssets: AssetAmountInterface;
+  bestTrade: Trade | [];
+  bestTradeWithSlippageTolerance: Trade | [];
```

Comment:
do you really need to save `bestTrade` and `bestTradeWithSlippageTolerance` inside form?  user do not input them

Reply:
yes, because form and submit handler in different component